bareon_api/api/controllers/partitioning.py

```diff
@@ -83,6 +83,11 @@ class VGController(SimpleRestController):
     collection = models.LVS
 
 
+# TODO figure out the terminology, the handler
+# returns not only partitionins, but also
+# volume groups, logical volumes etc. Some
+# time ago we had an idea to call these entities
+# `spaces` not to confuse it with real partitions
```

Comment:
This controller actually returns only Partitions. AFAIK you meant `PartitioningController`